index.html

```diff
@@ -1,11 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Product Landing Page</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="landingPage.css">
 </head>
 <body>
-  
+  <!-- Navigation bar -->
+    <header>
+        <nav>
+            <a href="#Home"><img src="logo.png" alt="Home"></a>
+            <ul>
+                <li><a href="#Home">Home</a></li>
+                <li><a href="#Shop">Shop</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <div class="hero">
+      <img id="hero-hats" src="heroHats.jpeg" alt="Hero Hats">
+      <div class="hero-text">
+        <h1>Tip Top Hats</h1>
+        <p>All handcrafted in Australia</p>
+        <button>Shop Now</button>
+      </div>
+    </div>
+
+    <!-- Description of website -->
+    <section class="description">
+      <p>Tip Top Hats was founded in 1534 by an Australian farmer, Henry Tiptop. The original hat was crafted from kangaroo hide. The techniques of our hat crafting has since been perfected and past down for many generations. We have even branched out into a new line of carbon-free and vegan-friendly hats to accommodate all lifestyles.</p>
+    </section>
```

Comment:
**[Change Request]**
`<section>` requires a heading -- you can always check your html with a [HTML validator](https://validator.w3.org/#validate_by_input).
For this part a better way to organise the HTML would be:
```html
    <main>
        <section class="hero">
            <img id="hero-hats" src="heroHats.jpeg" alt="Hero Hats">
          <div class="hero-text">
            <h1>Tip Top Hats</h1>
            <span>All handcrafted in Australia</span>
            <button>Shop Now</button>
          </div>
      
        <p>Tip Top Hats...</p>
       </section>
    </main>
```
...I don't think there are Europeans in Australia in 1534 LOL 😂 
Anyway after you made the changes the `css` would unfortunately change, but I think we should go through that together. At the moment your button is covering your description.
<img src="https://github.com/debccheng/product-landing-page/assets/47295273/87b06678-1583-4017-b235-d71351a59f1c" width="800px">

Reply:
Not really sure what this "<section> requires a heading" means. Hero text and button no longer out of position - fixed

Reply:
<img width="371" alt="Screenshot 2023-11-11 at 1 37 44 am" src="https://github.com/debccheng/product-landing-page/assets/101181556/9e13e691-b762-49fd-b322-c7788c6d33fc">

Nvm, my overlay breaks when I change the dimensions smaller (i.e. iphone se). A bit stuck with fixing this